From: Siddhartha Sahu <sh.siddhartha@gmail.com>
Date: Tue, 06 Oct 2015 13:51:52 +0000
Subject: Include env vars from sourced files on pre-startup
X-Git-Url: http://quickgit.kde.org/?p=plasma-workspace.git&a=commitdiff&h=f869daca8244131f6b452e2c15b4dee5903ff768
---
Include env vars from sourced files on pre-startup

REVIEW: 125323
---


--- a/startkde/startkde.cmake
+++ b/startkde/startkde.cmake
@@ -206,7 +206,7 @@
 # Add /env/ to the directory to locate the scripts to be sourced
 for prefix in `echo $scriptpath`; do
   for file in "$prefix"/env/*.sh; do
-    (test -r "$file" && . "$file") || :
+    test -r "$file" && . "$file" || true
   done
 done
 

